quantecon/kalman.py

```diff
@@ -19,7 +19,7 @@ class Kalman:
 
     .. math::
 
-        x_{t+1} = A x_t + C w_{t+1}
+        x_{t+1} = A x_t + C w_{t+1} \\
```

Comment:
@natashawatkins should these be aligned?

Reply:
No I wouldn't say so - they're different equations

Reply:
Not a big deal but it would be natural to align these equations on the equals sign.

Reply:
I ask as they will be right aligned (by default)

![image](https://user-images.githubusercontent.com/8263752/31796966-95e4380a-b577-11e7-8ce9-0724d395e860.png)

Reply:
I think it will make it quite messy when you view the docstring in jupyter for example (which simply displays the raw string and not latex)

I can't see it done anywhere else in the docs

You would need to use the align environment right?

Reply:
mmm, good point.  perhaps it's best as is.  probably not worth spending more time on.

Reply:
I was just about to say the same thing :).